app/(product)/private-cloud/(product)/product/[licencePlate]/@decision/page.tsx

```diff
@@ -57,6 +57,12 @@ export default function RequestDecision({ params }: { params: { licencePlate: st
     values: { humanCommnet: '', decision: '', ...data?.requestedProject },
   });
 
+  useEffect(() => {
+    if (!session?.user?.roles?.includes('admin')) {
```

Comment:
We we use `session?.isAdmin` as it is available now?

Reply:
Ah yes! Will refactor